apps/desktop-wallet/src/modals/TransactionDetailsModal.tsx

```diff
@@ -77,274 +76,282 @@ const TransactionDetailsModal = ({ transaction, onClose }: TransactionDetailsMod
-                  <ActionLinkStyled
-                    onClick={() => handleShowAddress(transaction.address.hash)}
-                    key={transaction.address.hash}
-                  >
-                    <AddressBadge addressHash={transaction.address.hash} truncate withBorders />
-                  </ActionLinkStyled>
-                </AddressList>
-              ) : (
-                <IOList
-                  currentAddress={transaction.address.hash}
-                  isOut={false}
-                  outputs={transaction.outputs}
-                  inputs={transaction.inputs}
-                  timestamp={transaction.timestamp}
-                  linkToExplorer
-                />
-              )}
-            </DetailsRow>
-            <DetailsRow label={t('To')}>
-              {direction !== 'out' ? (
-                <AddressList>
-                  <ActionLinkStyled
-                    onClick={() => handleShowAddress(transaction.address.hash)}
-                    key={transaction.address.hash}
-                  >
-                    <AddressBadge addressHash={transaction.address.hash} withBorders />
-                  </ActionLinkStyled>
-                </AddressList>
-              ) : (
-                <IOList
-                  currentAddress={transaction.address.hash}
-                  isOut={direction === 'out'}
-                  outputs={transaction.outputs}
-                  inputs={transaction.inputs}
-                  timestamp={transaction.timestamp}
-                  linkToExplorer
-                />
-              )}
-            </DetailsRow>
-          </>
-        )}
-        <DetailsRow label={t('Status')}>
-          {transaction.scriptExecutionOk ? (
-            <Badge color={theme.global.valid}>
-              <span tabIndex={0}>{t('Confirmed')}</span>
-            </Badge>
-          ) : (
-            <Badge color={theme.global.alert}>
-              <span tabIndex={0}>{t('Script execution failed')}</span>
-            </Badge>
-          )}
-        </DetailsRow>
-        <DetailsRow label={t('Timestamp')}>
-          <span tabIndex={0}>{formatDateForDisplay(transaction.timestamp)}</span>
-        </DetailsRow>
-        {lockTime && (
-          <DetailsRow label={lockTime < new Date() ? t('Unlocked at') : t('Unlocks at')}>
-            <span tabIndex={0}>{formatDateForDisplay(lockTime)}</span>
-          </DetailsRow>
-        )}
-        <DetailsRow label={t('Fee')}>
-          <Amount tabIndex={0} value={BigInt(transaction.gasAmount) * BigInt(transaction.gasPrice)} fullPrecision />
-        </DetailsRow>
-        <DetailsRow label={t('Total value')}>
-          <Amounts>
+          </TransactionType>
+          <AmountWrapper tabIndex={0}>
             {tokensWithSymbol.map(({ id, amount, decimals, symbol }) => (
               <AmountContainer key={id}>
                 <Amount
                   tabIndex={0}
                   value={amount}
-                  fullPrecision
```

Comment:
We don't want that anymore? What's the reasoning?

Reply:
It's still here. Look bellow in the new diff ;)